README.md

```diff
@@ -2,6 +2,14 @@
 [![GitHub license](https://img.shields.io/badge/license-LGPL-blue.svg)](https://raw.githubusercontent.com/PedestrianDynamics/jupedsim/master/LICENSE)
 ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/jupedsim)
 ![PyPI - Version](https://img.shields.io/pypi/v/jupedsim)
+[![Build CI wheels](https://github.com/PedestrianDynamics/jupedsim/actions/workflows/cibuildwheel.yml/badge.svg)](https://github.com/PedestrianDynamics/jupedsim/actions/workflows/cibuildwheel.yml)
```

Comment:
I think we do not need this badge as we require working builds for merges. Also the presented name is not understandable for people not involved in the project, I.e. what is “build ci wheels”supposed to mean. 

I would just drop it.